modules/nested-accordion/assets/scss/frontend.scss

```diff
@@ -33,26 +34,27 @@
 			position: relative;
 
 			&:not(:last-child) {
-				margin-bottom: var(--n-accordion-item-title-space-between);
+				margin-block-end: var(--n-accordion-item-title-space-between);
 			}
 
 			:where( & > .e-con ) {
-				border: var(--n-accordion-border-width) var(--n-accordion-border-style) var(--n-accordion-border-color);
+				border: var( --n-accordion-border-width ) var( --n-accordion-border-style ) var( --n-accordion-border-color );
 			}
 
 			&-title {
 				display: flex;
 				flex-direction: row;
 				list-style: none;
-				padding: var(--n-accordion-padding);
-				border-width: var(--n-accordion-border-width);
-				border-color: var(--n-accordion-border-color);
-				border-style: var(--n-accordion-border-style);
-				justify-content: var(--n-accordion-title-justify-content);
-				border-radius: var(--n-accordion-border-radius);
-				flex-grow: var(--n-menu-title-flex-grow);
-				gap: var(--n-accordion-icon-gap);
-				color: var(--n-accordion-title-normal-color);
+				padding-block: var( --n-accordion-padding-block,  var( --n-accordion-padding ) );
+				padding-inline: var( --n-accordion-padding-inline, var( --n-accordion-padding ) );
```

Comment:
the shorthand are supported by Safari 14.1, Elementor supports Safari 14.

ref: https://caniuse.com/mdn-css_properties_padding-block

Reply:
@rami-elementor Thanks for pointing that out, I've removed all shorthand logical properties